node/src/command.rs

```diff
@@ -23,7 +23,7 @@ use node_template_runtime::Block;
 
 impl SubstrateCli for Cli {
 	fn impl_name() -> String {
-		"Substrate Node".into()
+		"S-O Substrate Node".into()
```

Comment:
I'm 100% sure where this is being displayed, but can we give it a more descriptive name?

Reply:
I was using it to test, but I'll revert to original state